app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -194,6 +194,7 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
        $scope.emsCommonModel.emstype == "openstack" && $scope.emsCommonModel.default_hostname ||
        $scope.emsCommonModel.emstype == "scvmm" && $scope.emsCommonModel.default_hostname ||
        $scope.emsCommonModel.emstype == "openstack_infra" && $scope.emsCommonModel.default_hostname ||
+       $scope.emsCommonModel.emstype == "nuage_network"  && $scope.emsCommonModel.default_hostname ||
```

Comment:
@AparnaKarve Aparna, we are not that conversant with the UI pieces. Could you please elaborate that how can we get the hostname field and subsequently validation enabled here? Please let us know and we will incorporate the changes.